packages/terra-functional-testing/lib/config/wdio.conf.js

```diff
@@ -63,7 +64,7 @@ exports.config = {
   // Define all options that are relevant for the WebdriverIO instance here
   //
   // Level of logging verbosity: trace | debug | info | warn | error | silent
-  logLevel: 'info',
+  logLevel: 'silent',
```

Comment:
Why change to silent? Just curious. I have no objections to the change

Reply:
The default log level of `info` is kind enough to print out the entire command being run during `browser.execute`. In this case, ~20,000 lines of minified axe-code. The existing configuration is set to silent, I suspect for similar reasons:

https://github.com/cerner/terra-toolkit-boneyard/blob/master/config/wdio/wdio.conf.js#L72

Reply:
oh fun.... did warn or error log these as well?